full/coupler_main.F90

```diff
@@ -313,47 +313,11 @@ program coupler_main
   !--- F90 module for OpenMP
   use omp_lib
 
-  use constants_mod,           only: constants_init
+  use FMS, version_fms=>version, status_fms=>status
 
-  use fms_affinity_mod,        only: fms_affinity_init, fms_affinity_set
+  !< Can't get rid of this until fms_io is no longer used at all
+  use fms_io_mod,              only: fms_io_exit, write_version_number
```

Comment:
Once write_version_number is moved to fms_mod and is publicized via 'use FMS', this will need to be modified.